api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -171,25 +167,7 @@ public void setUp() throws MessagingException {
             fireCloudService,
             directoryService,
             cloudStorageService,
-            leonardoNotebooksClient,
```

Comment:
nit: if you still have appetite for refactoring, consider autowiring this. I believe we shouldn't need to use `Providers.of()` if we're setting up the test configuration properly.